src/sentry_utils.c

```diff
@@ -339,3 +340,26 @@ sentry__dsn_get_attachment_url(
     sentry__stringbuilder_append(&sb, "/attachments/");
     return sentry__stringbuilder_into_string(&sb);
 }
+
+void
+sentry__utcnow(struct tm *tm_out)
+{
+    time_t now;
+    time(&now);
+    struct tm *rv;
+#ifdef SENTRY_PLATFORM_WINDOWS
+    rv = gmtime(&now);
+#else
+    struct tm buf;
+    rv = gmtime_r(&now, &buf);
```

Comment:
`gmtime` is not threadsafe on non windows platforms, so we need to use `gmtime_r` here.